### PR TITLE
[keybinding] Align more Electron/browser keybindings with VS Code

### PR DESCRIPTION
**Pull request by jankeromnes**
#### What it does

- [x] Add <kbd>Ctrl/Cmd</kbd> + <kbd>N</kbd> = `File` > `New File`
- ~~Fix <kbd>Ctrl/Cmd</kbd> + <kbd>O</kbd> = `File` > `Open...` (currently discussed in #6633)~~ (EDIT: let's do that later)
- [x] Align <kbd>Ctrl/Cmd</kbd> + <kbd>W</kbd> with VS Code on all OSes

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by jankeromnes** [comment]
Summary of relevant keybindings in VS Code:

| VS Code | Windows | Mac OS | Linux |
| --- | --- | --- | --- |
| New File | | | |
| Close Editor | <kbd>Ctrl</kbd> + <kbd>F4</kbd> | <kbd>Cmd</kbd> + <kbd>W</kbd> | <kbd>Ctrl</kbd> + <kbd>W</kbd> |
| Close Window | <kbd>Ctrl</kbd> + <kbd>W</kbd> | <kbd>Shift</kbd> + <kbd>Cmd</kbd> + <kbd>W</kbd> | <kbd>Ctrl</kbd> + <kbd>Q</kbd> |
| Settings | <kbd>Ctrl</kbd> + <kbd>,</kbd> | <kbd>Cmd</kbd> + <kbd>,</kbd> | <kbd>Ctrl</kbd> + <kbd>,</kbd> |
| Save As | | | |

Summary of keybindings in Theia with this PR:

| Theia (Electron) | Windows | Mac OS | Linux |
| --- | --- | --- | --- |
| New File | | | |
| Close Editor | | | |
| Close Window | | | |
| Preferences | | | |
| Save As | | | |

| Theia (Chrome) | Windows | Mac OS | Linux |
| --- | --- | --- | --- |
| New File | | | |
| Close Editor | | | |
| Close Window | | | |
| Preferences | | | |
| Save As | | | |


| Theia (Firefox) | Windows | Mac OS | Linux |
| --- | --- | --- | --- |
| New File | | | |
| Close Editor | | | |
| Close Window | | | |
| Preferences | | | |
| Save As | | | |

**Comment by jankeromnes** [comment]
Let's tackle <kbd>Ctrl/Cmd</kbd> + <kbd>O</kdb> in a follow-up PR, so that we can move forward here.

**Comment by jankeromnes** [comment]
I didn't actually test on Linux, nor on Electron Windows yet, but this is ready for review now.

File: packages/workspace/src/browser/workspace-frontend-contribution.ts [hunk 1]
```diff
@@ -141,6 +141,12 @@ export class WorkspaceFrontendContribution implements CommandContribution, Keybi
     }
 
     registerKeybindings(keybindings: KeybindingRegistry): void {
+        if (this.isElectron()) {
+            keybindings.registerKeybinding({
```
Comment (svenefftinge) [inline packages/workspace/src/browser/workspace-frontend-contribution.ts]: wouldn't it make sense to register the command for browser as well? In that case use `shift` in addition?

Comment (svenefftinge) [reply packages/workspace/src/browser/workspace-frontend-contribution.ts]: Just figured that will create a private browser in chrome. So maybe use `alt`.

Comment (jankeromnes) [reply packages/workspace/src/browser/workspace-frontend-contribution.ts]: `alt+n` could work. Thanks for suggestion!

**Comment by jankeromnes** [comment]
Thank you Sven! Please also merge the PR if you have time, since I can't do that myself yet.

**Pull request merged**